src/integrationTest/java/uk/gov/hmcts/darts/ws/AddAudioWebServiceTest.java

```diff
@@ -58,10 +61,15 @@ class AddAudioWebServiceTest extends IntegrationBase {
     @MockBean
     private TokenValidator validator;
 
-
     @Value("${darts-gateway.add-audio.fileSizeInMegaBytes}")
     private long maxByteSize;
```

Comment:
Can be changed to Datasize like darts api

Reply:
changed to DataSize